### PR TITLE
Giving Soteria department account access.

### PR DESCRIPTION
**Pull request by MSieghardt**
Scientists and Roboticists gain access to departmental funds to be able to buy / sell stuff now. 

Tested it on my server. Works just fine.

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Veto

**Comment by Trilbyspaceclone** [comment]
This does not help RP
This makes CRO useless form their own departments, allows for the sapping of funds and hard to recharge.

**Comment by ThePainkiller** [comment]
Agreed, while this might be good for those shifts in which there's lowpop enough not to have a CRO, this beats the whole purpose of having one at the same time, and there is no way this isn't gonna be abused to buy non-science stuff since you're trusting non-command science players with easy access to a lot of money. There's a reason departamental funds are linked to the current head of staff's one...

**Comment by Trilbyspaceclone** [comment]
Theirs MANY issues with how the balance of funds work for SI sci.
As Sci your job isnt to sell stuff to just anyone, is not to make a proffit. Those are side goals, not the objective. Your their for doing rnd work, form plants to advanced gear, making a proffit is second objective. 
If we go down this road then - Why do propis not have access to banking?
Medical, blackshield and marshal officers are all valid questions on they would not get access to the bank when its a job of a head of staff to do financing and fund their workers with gear/credits and help they require.
Cargo techs get it cuz its their JOB to handle transactions
Spec officer gets it cuz its their JOB to handle sales of fire arms and trade in guns
Miners get it cuz its their JOB to sell materals using the consoles
Janitors dont get it, its outside their basic job, same goes with every other job we have.

File: code/game/jobs/job/science.dm [hunk 1]
```diff
@@ -68,6 +68,7 @@
 	spawn_positions = 5
 	supervisors = "the Chief Research Overseer"
 	difficulty = "Medium."
+	department_account_access = TRUE
```
Comment (Trilbyspaceclone) [inline code/game/jobs/job/science.dm:71]:
```suggestion
	department_account_access = FALSE
```
Middle-gound is that robotics get access as they are MEANT to be a implanter wielding and dealing person selling and buying mech gear/selling stuff

**Comment by MSieghardt** [comment]
So, let me respond to Painkiller first:

Cargo Technicians, who require 0 qualifications, have access to departmental funds, but doctors and researchers, people with actual degrees and entrusted with extremely dangerous stuff, can't be trusted?

They need the funds to be able to actually make things fun. I have been playing quite a bit as of late and Soteria just doesn't have the money to pay for my materials if I'm a miner. Bartering is dumb, we're not a medieval commune. It makes 100% sense for researchers to be able to draw upon funds.

Abused? You do know that there's a paper trail, right? Any withdrawal / deposit is traced in the bank log. If you think embezling is being done, check it and have them charged with theft.

**Comment by Trilbyspaceclone** [comment]
No for normale sci, middle ground for Robotics.
It comes into `Is my job handling money?`. Otherwise we should give everyone in every department the access to funds

**Review by Trilbyspaceclone — APPROVED** [review]
Good middle ground

**Pull request merged**